src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingLumberjack.java

```diff
@@ -426,43 +422,45 @@ private void bonemealFungi()
             return;
         }
         final int modifier = Math.max(0, Math.min(FUNGI_MODIFIER, 100));
-        for (final BlockPos pos : netherTrees)
+        for (Iterator<BlockPos> iterator = netherTrees.iterator(); iterator.hasNext(); )
```

Comment:
why not while iterator has next? and put the iterator at the top?

Reply:
doesnt matter its equivalent xD